cf-testd/cf-testd.c

```diff
@@ -202,23 +188,22 @@ bool CFTestD_TLSSessionEstablish(ServerConnectionState *conn)
     /* Send/Receive "CFE_v%d" version string, agree on version, receive
        identity (username) of peer. */
     char username[sizeof(conn->username)] = "";
-    bool id_success = ServerIdentificationDialog(conn->conn_info,
-                                                 username, sizeof(username));
+    bool id_success                       = ServerIdentificationDialog(
+        conn->conn_info, username, sizeof(username));
```

Comment:
WTH?

Reply:
This looks like there was another assignment here at some point, and it was trying to align the equal signs, but @craigcomstock moved that assignment and didn't run clang-format again.

Reply:
there is an assignment right above this. The line is too long without some breaking.

I would maybe do this: 
```    
    char username[sizeof(conn->username)] = "";
    bool id_success = ServerIdentificationDialog(
        conn->conn_info, 
        username, 
        sizeof(username)
    );
```

Reply:
> @craigcomstock Put an empty line between the 2 assignments and it won't align them.

That's exactly stuff I would like to avoid doing.